frontend/src/modules/projectinfo/components/ProjectInfo.js

```diff
@@ -1,6 +1,7 @@
 /* @flow */
 
 import * as React from 'react';
+import { Localized } from 'fluent-react';
```

Comment:
Oh, and this is incorrect. You might have started working on this a few days ago, and we changed the way we import fluent libraries since then. Please get the latest changes from master to see the correct way of importing.

Reply:
Done